src/libs/ReportUtils.ts

```diff
@@ -3275,6 +3275,40 @@ function buildOptimisticExpenseReport(chatReportID: string, policyID: string, pa
     return expenseReport;
 }
 
+function getIOUSubmittedMessage(report: OnyxEntry<Report>) {
+    const policy = getPolicy(report?.policyID);
+    const ownerPersonalDetails = getPersonalDetailsForAccountID(policy?.submitsTo ?? 0);
+    let submittedToDisplayName: string;
+    if (ownerPersonalDetails?.accountID === currentUserAccountID) {
+        submittedToDisplayName = 'yourself';
+    } else {
+        submittedToDisplayName = `${ownerPersonalDetails.displayName ?? ''}${ownerPersonalDetails.displayName !== ownerPersonalDetails.login ? ` (${ownerPersonalDetails.login})` : ''}`;
+    }
```

Comment:
Let's do this since it looks much more simple

```suggestion
    let submittedToDisplayName = `${ownerPersonalDetails.displayName ?? ''}${ownerPersonalDetails.displayName !== ownerPersonalDetails.login ? ` (${ownerPersonalDetails.login})` : ''}`;
    if (ownerPersonalDetails?.accountID === currentUserAccountID) {
        submittedToDisplayName = 'yourself';
    }
```